scripts/schemas/twister/testsuite-schema.yaml

```diff
@@ -8,200 +8,209 @@
 #
 # The original spec comes from Zephyr's twister script
 #
+schema;scenario-schema:
+  type: map
+  # has to be not-required, otherwise the parser gets
+  # confused and things it never found it
+  required: false
+  mapping:
+    "arch_exclude":
+      type: any
+      required: false
+    "arch_allow":
+      type: any
+      required: false
+    "testcases":
+      type: seq
+      required: false
+      sequence:
+        - type: str
+    "build_only":
+      type: bool
+      required: false
+    "build_on_all":
+      type: bool
+      required: false
+    "depends_on":
+      type: any
+      required: false
+    "extra_args":
+      type: any
+      required: false
+    "extra_configs":
```

Comment:
now `extra_configs` is allowed in the common section ?

Reply:
yes